ThermalTalk/ThermalTalk.csproj

```diff
@@ -5,7 +5,7 @@
     <Version>2.1.1.0</Version>
```

Comment:
Let's move to v3

Reply:
And lets add a section about v3 in the readme as well as a statement on what .NET framework users should do in order to use thermaltalk (i.e. use v2)